sql/data/[User].csv

```diff
@@ -0,0 +1,7 @@
+id,name
+,Emma
+,Malchau
+,Jepp
+,Jacob
+,Freja
+,Josefine
```

Comment:
What is this used for?

Reply:
Old Docker test. It should not be merged into main :) Might certainly explode.